src/c++/perf_analyzer/test_concurrency_manager.cc

```diff
@@ -592,20 +554,57 @@ TEST_CASE("Concurrency - Check setting of InitSharedMemory")
   bool is_decoupled = false;
   bool use_mock_infer = true;
 
+  const std::string json_str{R"(
```

Comment:
This shouldn't be needed for this test.

Regardless, I think this whole test can go away. All we are testing here now is a single bool in a mock class. I think this behavior is now fully encapsulated in the `Concurrency - Shared memory methods` test.

Reply:
removed